engine/src/main/java/org/terasology/utilities/subscribables/AbstractGeneralSubscribable.java

```diff
@@ -18,24 +18,18 @@
 import java.beans.PropertyChangeListener;
 import java.beans.PropertyChangeSupport;
 
-/**
- * {@inheritDoc}
- */
+
```

Comment:
It's great that you decided to remove these redundant javadocs.

However, note that every method now has 2 newlines (in this case, 20 and 21) perceding it. We usually like 1 newline. Can you please remove the extra newline for all the javadocs you deleted?